packages/odyssey-eslint/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@okta/odyssey-eslint",
+  "version": "0.0.0",
```

Comment:
Back in the early days of this repo, we leveraged lerna to unify our versioning. Are we now managing each package independently to have more control over versioning?

Reply:
No, not for Odyssey at the moment. We plan to revisit this discussion in a few months however I have some thoughts